src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -129,8 +129,11 @@ export class BrowsingContextProcessor {
       // debugger  and ignore them.
       targetCdpClient
```

Comment:
https://github.com/GoogleChromeLabs/chromium-bidi/blob/a09dc1c3132a074710632decb05f6cbc8cac29b7/.eslintrc.js#L65

I deactivated it because (IIRC) it was too intrusive and @sadym-chromium disliked it. Let me see if I can re-enable it.

Reply:
There are currently 4 violations, if we decide to enable it today:

```
🏃 [eslint] Running command "eslint --cache --ext .js,.ts --fix ."

/Users/tperrotta/Projects/chromium-bidi/src/bidiMapper/BidiServer.ts
  44:5  warning  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/tperrotta/Projects/chromium-bidi/src/bidiMapper/domains/context/browsingContextProcessor.ts
  130:7  warning  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/tperrotta/Projects/chromium-bidi/src/cdp/cdpConnection.ts
  109:7  warning  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises

/Users/tperrotta/Projects/chromium-bidi/src/utils/processingQueue.ts
  57:13  warning  Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
```

Reply:
This fix corresponds to the third one.

Reply:
let's enable it since otherwise unhandled promises start causing troubles...